aspnet/getting-started.rst

```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
```

Comment:
Are you 100% sure https://microsoft.com/net/core will work? you could use a fwlink. I'm guessing we'll want to update this to a dedicated install page.